olive/passes/openvino/quantization.py

```diff
@@ -97,3 +103,38 @@ def _run_for_config(self, model: OpenVINOModel, config: Dict[str, Any], output_m
         openvino_model = OpenVINOModel(model_path)
 
         return openvino_model
+
+    def _create_dataloader(self, common_dataloader):
+        """
+        Create an openvino.tools.pot.api.DataLoader instance from a common dataloader.
+        """
+        try:
+            from openvino.tools.pot.api import DataLoader
+        except ImportError:
+            raise ImportError("Please install olive-ai[openvino] to use OpenVINO pass")
+
+        class _OVDataloader(DataLoader):
+            def __init__(self, dataloader):
+                self.dataloader = dataloader
```

Comment:
>self.dataloader [](http://example.com/codeflow?start=16&length=15)

it seems the data member is not needed since all the data is already transformed to data/label?

Reply:
Removed.